04 - Guides, Workflows, & Courses/Guides/Using Visual Studio Code for Theme Development.md

```diff
@@ -0,0 +1,235 @@
+
+**More Info: [Column (Box) Selection](https://code.visualstudio.com/docs/editor/codebasics#_column-box-selection)**
+
+### Toggle Wrapping in Current File
+
+`Alt+Z` (Windows/Linux/MacOS)
+
+> 💡 **EXAMPLE**
+> ![[VSCode Guide - Toggle Wrapping.gif]]
+> If your file includes long lines of text e.g. Base64 embedded fonts/SVGs toggling Wrapping will make them remain on one line no matter the length or wrap the text around to the next line depending on the width of the File's window.
+
+**More Info: [Toggle Wrapping in Current File](https://code.visualstudio.com/docs/editor/codebasics#_how-do-i-turn-on-word-wrap)**
+
+### Rename Symbol
+
+Click on what you want to rename and press `F2` (Windows/Linux/MacOS)
+
+> 💡 **EXAMPLE**
+> ![[VSCode Guide - Rename Symbol.gif]]
+> To rename any text/variable instances, click on one of them and press `F2`. Change the name to what you want and it will rename all of the instances of the selected text/variable to what you typed.
+
+**More Info: [Rename Symbol](https://code.visualstudio.com/docs/editor/editingevolved#_rename-symbol)**
+
+## ⚙️Settings 
+You can access the Settings by pressing:
+
+`Ctrl+,` (Windows/Linux)
+`Cmd+,` (MacOS)
+
+The Settings can be changed independently for:
+
+- User (Global)
+- Current Workspace
+- Currently Open Folder
+
+For generic settings I would recommend setting them in the User and for specific settings to the workflow of the project setting them in the Workspace.
+
+Here is a list of useful Settings (You can search for them using the Search Bar at the top of the Settings window):
+
+### Files: Auto Save
+Using the `After Delay` option the files save after a set delay once any change is made to the file. You can adjust the delay time in the `Files: Auto Save Delay` setting to whatever you want. I have it set to `1000` milliseconds (1 second).
+
+### Editor: Font Family
+This will change the font used in the editor area of VSCode. The fonts are sourced from the ones installed on your Operating System. Use the name of the system font to set it. You can also adjust the size of the font using the `Editor: Font Size` setting.
+
+### Files: Exclude
+If there are any files or folders you don't want to show up in the File Explorer you can hide them by adding a new pattern.
+
+**More Info: [Settings](https://code.visualstudio.com/docs/getstarted/settings)**
+
+## 💾 Settings Sync
+If you happen to be working with VSCode across multiple devices it is a good idea to setup Settings Sync. It will synchronise settings, key bindings and installed extensions across your machines. You will need to use a Microsoft account or a GitHub account to sync.
+
+Simply click on the `Cog` icon in the bottom left corner and select `Turn On Settings Sync...`. You will be presented with the options for data to be synchronised. 
+
+**More Info: [Settings Sync](https://code.visualstudio.com/docs/editor/settings-sync)**
+
+## 🔌 Extensions
+Here is a list of useful Extensions:
```

Comment:
Here are a couple of suggestions:
Change Color Format - enables you to quickly convert colors into different color formats
SCSS IntelliSense - useful autocompletion for Sass variables, mixins, etc